packages/paste-core/components/badge/src/index.tsx

```diff
@@ -27,7 +27,7 @@ export const Badge = React.forwardRef<HTMLSpanElement, BadgeProps>(({variant, ch
         fontSize="fontSize20"
         fontWeight="fontWeightSemibold"
         lineHeight="lineHeight10"
-        maxWidth="min-content"
+        maxWidth="fit-content"
```

Comment:
Is this intentional?

Reply:
Yes it is, this was an error from before alas.

Reply:
OK, don't forget to include a changeset for it in that case